package.json

```diff
@@ -83,7 +83,9 @@
     "synth": "cdk --app \"ts-node --project=tsconfig.cdk.json ./deployment/bin/hassu\" synth",
     "destroy": "cdk --app \"ts-node --project=tsconfig.cdk.json ./deployment/bin/hassu\" destroy \"*\" --require-approval never",
     "pipeline:bootstrap": "ACCOUNT_ARN=$(ts-node --project=tsconfig.cdk.json deployment/bin/get-account-arn.ts) && cdk bootstrap --app \"ts-node --project=tsconfig.cdk.json ./deployment/bin/hassu-pipeline\" --cloudformation-execution-policies $ACCOUNT_ARN",
-    "deploy:pipeline": "cdk --app \"ts-node --project=tsconfig.cdk.json ./deployment/bin/hassu-pipeline\" deploy hassu-pipeline --require-approval never"
+    "deploy:pipeline": "cdk --app \"ts-node --project=tsconfig.cdk.json ./deployment/bin/hassu-pipeline\" deploy hassu-pipeline --require-approval never",
+    "swagger:codegen": "cross-env VELHO_ACCESS_TOKEN=$(ts-node --project=tsconfig.cdk.json ./tools/velho/authenticate.ts) npm run swagger:codegen:generate",
+    "swagger:codegen:generate": "cross-env-shell ./tools/velho/gradlew -p ./tools/velho"
```

Comment:
tehtäiskö näin:
* npm run generate ajaa kaikki generoinnit, eli npm-run-all lint:*
* nykyinen "amplify codegen" laitetaan "generate:api" nimelle
* "swagger:codegen" laitetaan "generate:velhoapi" nimelle ja siihen putkeen kaikki tarvittavat komennot
Nyt sit pelkkä "npm run generate" hoitaa kaiken mitä pitää.

Reply:
Kuulostaa hyvältä. Mietin noiden yhdistämistä kyllä tuossa työstäessäkin. Alkaa noita skriptejä ollakin tässä projektissa. :)

Reply:
Hetkinen. Mulla oli ongelmia laittaa samalle skriptille toi ympäristömuuttujan määrittelevä skripti ja tuo gradle taskin ajo. Kokeilen vielä, jos vaikka saisin pelittää.